index.html

```diff
@@ -70,5 +70,198 @@ <h2 class="first-title">I'm Peter <br/> Glad to see you! </h2>
 </div>
 </div>
 </section>
+<section class="card-works" id="portfolio">
+    <div class="card card-works-layout1">
+        <a href ="#">
+            <img src="/Images/SnapshootPortfolio1.png" alt="SnapshootPortfolio1" class="snapshot1">
+        </a>
+        <div class="card-works-content1">
+            <h3 class="tonic"> Tonic </h3>
+            <ul class="ul-frame1 flex">
+                <li class="client">CANOPY</li>
+                <li> <img src="/Images/Counter.png" alt="counter"/></li>
+                <li class="role"> Back End Dev</li>
+                <li> <img src="/Images/Counter.png" alt="counter"/></li>
+                <li class="year"> 2015 </li>
+            </ul>
+            <p class="description3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="grid">
+                <li class="tag html-tag">html</li>
+                <li class="tag css-tag">css</li>
+                <li class="tag js-tag">javascript</li>
+            </ul>
+            <div class="works-button ">
+                <button type="button" class="normal-button1">See Project</button>
+            </div>
+        </div>
+    </div>
+    <div class="card card-works-layout2">
+        <a href="#">
+            <img src="/Images/SnapshootPortfolio2.png" alt="SnapshootPortfolio2" class="snapshot2">
+        </a>
+        <div class="card-works-content2">
+            <h3 class="multi-post">Multi-Post Stories</h3>
+            <ul class="ul-frame1 flex">
+                <li class="client">CANOPY</li>
+                <li><img src="/Images/Counter.png" alt="counter"/></li>
+                <li class="role"> Back End Dev</li>
+                <li> <img src="/Images/Counter.png" alt="counter"/></li>
+                <li class="year"> 2015 </li>
+            </ul>
+            <p class="description3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="grid">
+                <li class="tag html-tag">html</li>
+                <li class="tag css-tag">css</li>
+                <li class="tag js-tag">javascript</li>
+            </ul>
+            <div class="works-button ">
+                <button type="button" class="normal-button1">See Project</button>
```

Comment:
- [ ] Please follow the figma design to implement button interaction stylings. The color for the hover and active state of the see Project button is different from the design.